src/cms.h

```diff
@@ -0,0 +1,35 @@
+#ifndef RM_CMS_H
+#define RM_CMS_H
+
+#include <stddef.h>
+
+#include "../contrib/xxhash.h"
+
+// #define REDIS_MODULE_TARGET
+
+#ifdef REDIS_MODULE_TARGET // should be in .h or .c
+#define CMS_CALLOC(count, size) RedisModule_Calloc(count, size)
+#define CMS_FREE(ptr) RedisModule_Free(ptr)
+#else
+#define CMS_CALLOC(count, size) calloc(count, size)
+#define CMS_FREE(ptr) free(ptr)
+#endif
+
+typedef __uint128_t long128;
+
+typedef struct
+{
+    size_t width;
+    size_t depth;
+    size_t *array;    
+    long128 counter; // might be used for top k results
```

Comment:
Not sure what this is for, but better leave out until it is needed.

Reply:
I thought to make space for a larger global counter but ended up not using.
Cleaned!